### PR TITLE
images: bump ubi to 8.3-297.1618432833 and ubi-minimal to 8.3-298.1618432845

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:** bump image versions

**Motivation for the change:** bugfix

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by jmrodri — APPROVED** [review]
/lgtm

File: images/ansible-operator/base.Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # It is built with dependencies that take a while to download, thus speeding
 # up ansible deploy jobs.
 
-FROM registry.access.redhat.com/ubi8/ubi:8.3-297
+FROM registry.access.redhat.com/ubi8/ubi:8.3-297.1618432833
```
Comment (jmrodri) [inline images/ansible-operator/base.Dockerfile:5]: It this right? the others have -298. this has -297.

Comment (estroz) [reply images/ansible-operator/base.Dockerfile:5]: ubi vs ubi-minimal

**Pull request merged**